doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,386 @@
+See :ref:`requirements` for information on the supported operating systems.
+
+.. _installing_vsc:
+
+.. rst-class:: numbered-step
+
+Install prerequisites
+*********************
+
+Depending on your preferred development environment, install the following tools:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      * The latest version of |VSC| for your operating system from the `Visual Studio Code download page`_.
+      * In |VSC|, the latest version of the |nRFVSC|.
+        For installation instructions, see `How to install the extension`_.
+      * Universal version of SEGGER J-Link.
+        This is required for SEGGER J-Link to work correctly with both Intel and ARM assemblies.
+
+   .. group-tab:: Command line
+
+      * The latest version of `nRF Util development tool`_, a unified command line utility for Nordic products.
+
+        .. note::
+            After downloading the nRF Util executable, move it to a directory that is in the system :envvar:`PATH`.
+            On macOS and Linux, the downloaded file also needs to be given execute permission by typing `chmod +x nrfutil` or by checking the checkbox in the file properties.
+
+      * Universal version of SEGGER J-Link.
+        This is required for SEGGER J-Link to work correctly with both Intel and ARM assemblies.
+
+.. _gs_installing_toolchain:
+.. _gs_installing_tools:
+
+.. rst-class:: numbered-step
+
+Install the |NCS| toolchain
+***************************
+
+The |NCS| :term:`toolchain` includes the Zephyr SDK and then adds on top of it tools and modules required to build |NCS| samples and applications.
+These include the :ref:`required tools <requirements_toolchain_tools>`, the Python dependencies, and the :ref:`GN tool <ug_matter_gs_tools_gn>` for creating :ref:`ug_matter` applications.
+
+Depending on your preferred development environment, install the |NCS| toolchain:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage toolchains`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install Toolchain`.
+         The list of available toolchain versions appears in the |VSC|'s quick pick.
+      #. Select the toolchain version to install.
+         The toolchain version should match the |NCS| version you are going to work with.
+         If you have received a custom URL for installing toolchain, you can provide it using the :guilabel:`Change Toolchain Index` button in the quick pick's header.
+         The toolchain installation starts in the background, as can be seen in the notification that appears.
+
+      When you install the toolchain for the first time, the installed version is automatically selected for your project.
+
+   .. group-tab:: Command line
+
+      1. Open a command prompt window.
```

Comment:
I think "command prompt" is a Windows specific term, "terminal" is a more universal term (also, nrfutil doesn't officially support command prompt, only `bash`, `zsh` & `pwsh`, so it's extra unfortunate)